api/src/main/java/jakarta/data/spi/provider/package-info.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * <p>This package defines the Service Provider Interface (SPI) by which
+ * Jakarta Data providers can make themselves discoverable by Jakarta EE products
+ * for CDI dependency injection.</p>
```

Comment:
```suggestion
 * for CDI dependency injection, if you don't want to use CDI extension.</p>
```

Reply:
That wouldn't be accurate because it _does_ result in CDI extension being used (the one provided by the Jakarta EE product/runtime). I need to update it to say that. Thanks for spotting this. It's corrected in https://github.com/jakartaee/data/pull/130/commits/9cecee168fdcb791c8442dd7a82db2d72ecbacc5